pkgs/development/libraries/mesa/default.nix

```diff
@@ -149,6 +150,7 @@ self = stdenv.mkDerivation {
     libX11 libXext libxcb libXt libXfixes libxshmfence libXrandr
     libffi libvdpau libelf libXvMC
     libpthreadstubs openssl /*or another sha1 provider*/
+    zstd
```

Comment:
Why? (Since a previous commit already says it fixed the build.)

Reply:
It's used for shader caches if available, which should be a tiny bit faster than zlib.

Reply:
Added a comment to the commit too, so it doesn't get lost.

Reply:
Thanks, an explanation in the commit message was what I was after.